STRINGS/Pallindrome/C/Pallindrome.c

```diff
@@ -0,0 +1,31 @@
+#include <stdio.h>
+#include <string.h>
+
+int isPallindrome(char str[]){
+
+    int len = strlen(str);
+    int i;
+
+    for(i=0; i<(len/2); i++){
+        // comparing ith character with "Length-i-1"th character
+        if( str[i] != str[len-i-1] ){
+            return 0;
+        }
+    }
+    return 1;
+
+}
+
+void main(){
```

Comment:
# Use int main here

Reply:
Yup there should be int main according to our repo standard although right to write void but its not a good pracctice to right void main in c